R/temporalBehaviour.R

```diff
@@ -393,8 +393,8 @@ finalizeResult <- function(finalResult, result, product, points, isoT, indices,
 #' Monthly Weather Review, 134(4), 1102–1120. https://doi.org/10.1175/MWR3106.1
 #'
 #' @examples
+#' \dontrun{
```

Comment:
In CRAN, they request `\donttest` rather than `\dontrun`

Reply:
Done